ghostwriter/reporting/templates/snippets/cvss_ui_v4.html

```diff
@@ -1322,13 +1322,13 @@ <h4 title="">Subsequent System Impact Metrics</h4>
                         >
                             Not Defined (X)
                         </label>
-                        <input
-                            type="radio"
-                            name="cvssv4_MSC"
-                            id="cvssv4_MSC_undefined"
-                            value="undefined"
-                        />
-                        <label for="cvssv4_MSC_undefined" title=""></label>
+{#                        <input#}
+{#                            type="radio"#}
+{#                            name="cvssv4_MSC"#}
+{#                            id="cvssv4_MSC_undefined"#}
+{#                            value="undefined"#}
+{#                        />#}
+{#                        <label for="cvssv4_MSC_undefined" title="" style="display:none;"></label>#}
```

Comment:
Let me check what this is supposed to be... I generated most of the CVSSv4 HTML with a script, and must have missed this issue with it.

Reply:
Apparently there's just an empty space in FIRST's CVSS v4 calculator there, which likely had a corresponding empty object in the schema I used but didn't account for. I'd just delete the input here.